fuzz/fuzz_targets/chanmon_fail_consistency.rs

```diff
@@ -80,6 +80,7 @@ impl Writer for VecWriter {
 	}
 }
 
+static mut IN_RESTORE: bool = false;
```

Comment:
I feel a bit uncertain about a shared variable between nodes but well code isn't threaded so shouldn't be a problem..

Reply:
Indeed, its not threaded, so it should be fine to use the unsafe{} access. Should be faster than useless atomics.